src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(currentValue, value) {
+      return currentValue + value;
```

Comment:
Consider using the assignment operator `+=` to add the value to the current value. This will simplify the expression and follow the code style best practices.